backend/app/models/log_records.py

```diff
@@ -14,7 +14,8 @@ class LogRecords(db.Model):
     # TODO: replace open String fields with VarChar(NUM_CHARS)
     note = db.Column(db.String, nullable=False)
     tags = db.Column(db.ARRAY(db.String), nullable=True)
-    building = db.Column(db.String, nullable=False)
+    building_id = db.Column(db.Integer, db.ForeignKey("buildings.id"), nullable=False)
+    building = db.relationship("Buildings", back_populates="log_record")
```

Comment:
i added this like you asked but i'm not really sure if its doing anything lol... same with the other ones

Reply:
this looks good 👍